ios/RNSentry.m

```diff
@@ -140,19 +140,23 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSLog(@"Bridge call to: deviceContexts");
+    NSMutableDictionary<NSString *, id> *contexts = [NSMutableDictionary new];
     // Temp work around until sorted out this API in sentry-cocoa.
     // TODO: If the callback isnt' executed the promise wouldn't be resolved.
     [SentrySDK configureScope:^(SentryScope * _Nonnull scope) {
         NSDictionary<NSString *, id> *serializedScope = [scope serialize];
         // Scope serializes as 'context' instead of 'contexts' as it does for the event.
-        NSDictionary<NSString *, id> *contexts = [serializedScope valueForKey:@"context"];
+        NSDictionary<NSString *, id> *tempContexts = [serializedScope valueForKey:@"context"];
+        if (tempContexts != nil) {
+            [contexts addEntriesFromDictionary:tempContexts];
+        }
```

Comment:
`l`:
```suggestion
        contexts = [serializedScope valueForKey:@"context"];
        if (contexts == nil) {
            context = [NSDictionary new];
        }
```

Reply:
if I do that, I need to check the `contexts` nullability outside of the callback too, due to https://github.com/getsentry/sentry-react-native/pull/2002#discussion_r783057923